src/utils/Firebase/firebase.js

```diff
@@ -33,6 +33,13 @@ class Firebase {
 
   // EXAMPLE ENDPOINT
   getLesson = () => this.db.collection("custom_lesson");
+  getCurrLesson = deploymentAccountId =>
```

Comment:
can u comment what this does above it but otherwise an absolute :fire: endpoint thanks gene